src/component/buy.js

```diff
@@ -0,0 +1,17 @@
+
+import React from 'react';
```

Comment:
I think you should rename this component to "select" or "selectTicket" because at this point they are not actually buying the ticket so the component name is misleading